scripts/smart-wallet/deployWallet.ts

```diff
@@ -3,12 +3,13 @@
 //
 // When running the script with `npx hardhat run <script>` you'll find the Hardhat
```

Comment:
let's not call this script deployWallet.ts which was already existing for wallet depoyment

can create another one for paymaster deployment